components/BannerManager/BannerManager.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC } from 'react';
+import { Trans } from 'react-i18next';
+import styled from 'styled-components';
+import { utils as ethersUtils } from 'ethers';
+
+import useGetLiquidationData from 'queries/liquidations/useGetLiquidationDataQuery';
+
+import Banner, { BannerType } from 'sections/shared/Layout/Banner';
+import { LOCAL_STORAGE_KEYS } from 'constants/storage';
+import { ExternalLink } from 'styles/common';
+import { zeroBN } from 'utils/formatters/number';
+import { formatShortDateWithTime } from 'utils/formatters/date';
+
+const BannerManager: FC = () => {
+	const liquidationData = useGetLiquidationData();
+
+	const liquidationRatio = liquidationData?.data?.liquidationRatio ?? zeroBN;
+	const liquidationDeadlineForAccount =
+		liquidationData?.data?.liquidationDeadlineForAccount ?? zeroBN;
+
+	const liquidationRatioPercentage = liquidationRatio.isZero()
+		? 0
+		: 100 / Number(ethersUtils.formatEther(liquidationRatio.toString()));
+
+	if (!liquidationDeadlineForAccount?.isZero()) {
```

Comment:
i think we can remove the optional ```?``` here since the 

```
const liquidationDeadlineForAccount
```

always returns a value

Reply:
true dat